### ISSUE TITLE
Passwords with # are loaded until the # character

### ISSUE THREAD

**Issue by jveiga** [Spotifyd/spotifyd#527]
**Description**
When running spotifyd with a password containing `#` the process fails with BadCredentials.

*NB:  It works when loading the password through the cli*
**To Reproduce**
Change the password to contain a `#` character and run spotifyd.

**Expected behavior**
Spotifyd should start.

**Logs**
```bash
Loading config from "/XXXXX/.config/spotifyd/spotifyd.conf"
No proxy specified
Using software volume controller.
error sending packet Os { code: 65, kind: Other, message: "No route to host" }
Connecting to AP "gew1-accesspoint-b-6xhq.ap.spotify.com:443"
Caught panic with message: Authentication failed with reason: BadCredentials
```

**Compilation flags** 
- [ ] dbus_mpris
- [ ] dbus_keyring
- [ ] alsa_backend
- [x] portaudio_backend
- [ ] pulseaudio_backend
- [ ] rodio_backend

**Versions (please complete the following information):**
 - OS: Mac OS Catalina 10.15.3 
 - Spotifyd: tag `v0.2.24`
- cargo: 1.41.0 (626f0f40e 2019-12-03)

I changed the code to print the actual password rather than the placeholder in order to debug the BadCredentials error.

**Comment by SirWindfield** [comment]
Related: https://github.com/Spotifyd/spotifyd/issues/411

Will get fixed in v1.0 probably as I wanted to move to `toml` instead of `ini`, which would just eliminate behaviour like this.

**Comment by macpijan** [comment]
Hit on the same thing, thanks for the explaination

**Comment by SirWindfield** [comment]
Resolved, #571
